packages/unified-latex-to-pretext/libs/break-on-boundaries.ts

```diff
@@ -0,0 +1,114 @@
+import { env, m } from "@unified-latex/unified-latex-builder";
+import * as Ast from "@unified-latex/unified-latex-types";
+import { getNamedArgsContent } from "@unified-latex/unified-latex-util-arguments";
+import {
+    anyEnvironment,
+    anyMacro,
+    match,
+} from "@unified-latex/unified-latex-util-match";
+import { replaceNode } from "@unified-latex/unified-latex-util-replace";
+import {
+    splitOnMacro,
+    unsplitOnMacro,
+} from "@unified-latex/unified-latex-util-split";
+import { visit } from "@unified-latex/unified-latex-util-visit";
+
+/**
+ * Breaks up division macros into environments
+ */
+export function breakOnBoundaries(ast: Ast.Ast): void {
+    const divisions = [
```

Comment:
It would be good to restructure this to have an object with keys division names and values their new env. names.